pymarsys/contact.py

```diff
@@ -32,3 +32,290 @@ def create(self, contact, key_id=None, source_id=None):
             self.endpoint,
             payload=payload
         )
+
+    def create_many(self, contacts, key_id=None):
+        """
+        Create many contacts from a list of dictionaries.
+        :param contacts:  a list of key-value pairs which uniquely identify
+        the contact fields which will be created for the contact (e.g. a key
+        can be the email field ID (3), and its value is the email address of
+        the specific contact).
+        :param key_id: Key which identifies the contacts.
+        :return: Dictionary with a list of the ids of the created contacts.
+        """
+        payload = {
+            'contacts': contacts,
+        }
+        if key_id:
+            payload['key_id'] = key_id
+
+        return self.connection.make_call(
+            'POST',
+            self.endpoint,
+            payload=payload
+        )
+
+    def query(self,
+              return_,
+              field_id_dict=None,
+              limit=None,
+              offset=None,
+              excludeempty=None):
```

Comment:
Same as above. This endpoint expects 'excludeempty', not exclude_empty. So I just wanted to stay coherent with the variable names they use. But this can be changed of course.